docs/StardustDocs/topics/readSqlDatabases.md

```diff
@@ -81,24 +81,33 @@ val df = DataFrame.readSqlTable(dbConfig, tableName, 100)
 df.print()
 ```
 
-Find full example project [here](https://github.com/zaleslaw/KotlinDataFrame-SQL-Examples/).
+Find a full example project [here](https://github.com/zaleslaw/KotlinDataFrame-SQL-Examples/).
 
 ## Getting Started with Notebooks
 
 To use the latest version of the Kotlin DataFrame library 
-and a specific version of the JDBC driver for your database (MariaDB is used as an example below) in your Notebook, run the following cell.
+and a specific version of the JDBC driver for your database (MariaDB is used as an example below) in your Notebook,
+run the following two cells.
 
-```
-%use dataframe
+First of all, specify the version of the JDBC driver
 
+```
 USE {
     dependencies("org.mariadb.jdbc:mariadb-java-client:$version")
 }
 ```
 
+In the second, enable the `Kotlin DataFrame` library usage in the following cell.
```

Comment:
"In the second" looks weird, I'd rephrase the entire sentence to make it a bit more concise:

Next, import Kotlin DataFrame in the cell below